Makefile

```diff
@@ -11,6 +11,8 @@ install:
 	cd $(connectors_sdk_dir); make install
 	cd $(app_dir); make install
 
+install-agent: install
```

Comment:
Do we need this if they're equivalent?

Reply:
Supposedly not, but in reality it's not gonna do almost anything and I kept it for safety 🤷